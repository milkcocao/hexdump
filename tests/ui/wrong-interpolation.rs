use anyhow::{bail, Result};

fn main() -> Resu