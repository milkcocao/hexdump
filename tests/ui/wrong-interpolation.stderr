error: 1 positional argument in format string, but no arguments 